creator-node/sequelize/migrations/20200928233308-create-content-blacklist.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('ContentBlacklists', {
+      id: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+        primaryKey: true
```

Comment:
Unfortunately, this wont work because user ids & track ids can clash and the primary key here will enforce unique. An alternative would be to keep an auto incrementing primary key `id` and adding a separate blacklist_item_id or something like that that is non-unique

Reply:
discussed offline -- type is also a primaryKey so this will work as expected